src/BibleWell.App/BibleWell.App.csproj

```diff
@@ -13,8 +14,9 @@
         <PackageReference Include="Avalonia.Themes.Fluent" />
         <PackageReference Include="Avalonia.Fonts.Inter" />
         <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.-->
-        <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" />
-        <PackageReference Include="CommunityToolkit.Mvvm" />
-        <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
+        <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics"/>
+        <PackageReference Include="CommunityToolkit.Mvvm"/>
+        <PackageReference Include="Microsoft.Extensions.DependencyInjection"/>
+        <PackageReference Include="Microsoft.Maui.Essentials"/>
```

Comment:
Had to `<PackageReference Include="Microsoft.Maui.Essentials" Version="9.0.0"/>` here to get ti to build after running `workload restore`

Reply:
But this seems to stop iOS from building with the error:
`2>BibleWell.App.csproj: Error NU1008 : Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: Microsoft.Maui.Essentials`

Reply:
that's weird. maybe you have to install maui essentials

Reply:
`dotnet add package Microsoft.Maui.Essentials --version 9.0.0`